godot-core/src/builtin/meta/godot_convert/impls.rs

```diff
@@ -240,3 +240,21 @@ impl FromGodot for *const std::ffi::c_void {
         Some(via as Self)
     }
 }
+
+// mut void* is used by ScriptExtension::instance_create()
```

Comment:
In C, it's just `void*`, not `mut void*`.

You could move this up to line 226, before `// Other impls...`, then it should be clearer.
Also please end with a full stop 🙂

Reply:
fixed!